readthedocs/core/views/__init__.py

```diff
@@ -42,7 +42,7 @@ def get_context_data(self, **kwargs):
         """Add latest builds and featured projects."""
         context = super().get_context_data(**kwargs)
         context['featured_list'] = Project.objects.filter(featured=True)
-        context['projects_count'] = Project.objects.count()
+        context['projects_count'] = Project.objects.exclude(slug="spam").count()
```

Comment:
I think we want to have something like this:
https://github.com/readthedocs/readthedocs.org/blob/7be01d844b981635822d4df73f614ad2c3d71b8b/readthedocs/projects/views/public.py#L46